When I try to merge the branch t3chguy/M600 into my repo (which basically is a fork of Jyers Ender_3_V2_Rewrite with my own config files) I get this compile error:

_Compiling .pio\build\STM32F103RET6_creality\src\src\libs\nozzle.cpp.o
Marlin\src\lcd\dwin\e3v2\dwin.cpp: In function 'void Menu_Item_Handler(uint8_t, uint8_t, bool)':
Marlin\src\lcd\dwin\e3v2\dwin.cpp:702:27: error: 'ChangeFilament' was not declared in this scope
                 Draw_Menu(ChangeFilament);
                           ^~~~~~~~~~~~~~
Marlin\src\lcd\dwin\e3v2\dwin.cpp:702:27: note: suggested alternative: 'load_filament'
                 Draw_Menu(ChangeFilament);
                           ^~~~~~~~~~~~~~
                           load_filament
Marlin\src\lcd\dwin\e3v2\dwin.cpp:971:12: error: 'ChangeFilament' was not declared in this scope
       case ChangeFilament:
            ^~~~~~~~~~~~~~
       case ChangeFilament:
            ^~~~~~~~~~~~~~
            load_filament
*** [.pio\build\STM32F103RET6_creality\src\src\lcd\dwin\e3v2\dwin.cpp.o] Error 1_

--- a/Marlin/src/lcd/dwin/e3v2/dwin.cpp
+++ b/Marlin/src/lcd/dwin/e3v2/dwin.cpp
@@ -627,7 +627,11 @@ int Get_Menu_Size(uint8_t menu) {
     case Info:
       return 0;
     case Tune:
-      return 8;
+      #if ENABLED(FILAMENT_LOAD_UNLOAD_GCODES)
+        return 9;
+      #else
+        return 8;
+      #endif
   }
   return 0;
 }
@@ -704,7 +708,11 @@ void Menu_Item_Handler(uint8_t menu, uint8_t item, bool draw/*=true*/) {
         #if ENABLED(ADVANCED_PAUSE_FEATURE)
           case 8: // Change Filament
             if (draw) {
-              Draw_Menu_Item(row, ICON_ResumeEEPROM, (char*)"Change Filament", true);
+              #if ENABLED(FILAMENT_LOAD_UNLOAD_GCODES)
+                Draw_Menu_Item(row, ICON_ResumeEEPROM, (char*)"Change Filament", true);
+              #else
+                Draw_Menu_Item(row, ICON_ResumeEEPROM, (char*)"Change Filament");
+              #endif
             } else {
               #if ENABLED(FILAMENT_LOAD_UNLOAD_GCODES)
                 Draw_Menu(ChangeFilament);
@@ -987,7 +995,7 @@ void Menu_Item_Handler(uint8_t menu, uint8_t item, bool draw/*=true*/) {
             break;
           case 1: // Load Filament
             if (draw) {
-              Draw_Menu_Item(row, ICON_WriteEEPROM, (char*)"Load Filament", true);
+              Draw_Menu_Item(row, ICON_WriteEEPROM, (char*)"Load Filament");
             } else {
               Popup_Window_LoadFilament();
               gcode.process_subcommands_now_P(PSTR("M701"));
@@ -997,7 +1005,7 @@ void Menu_Item_Handler(uint8_t menu, uint8_t item, bool draw/*=true*/) {
             break;
           case 2: // Unload Filament
             if (draw) {
-              Draw_Menu_Item(row, ICON_ReadEEPROM, (char*)"Unload Filament", true);
+              Draw_Menu_Item(row, ICON_ReadEEPROM, (char*)"Unload Filament");
             } else {
               Popup_Window_LoadFilament(true);
               gcode.process_subcommands_now_P(PSTR("M702"));
@@ -1007,7 +1015,7 @@ void Menu_Item_Handler(uint8_t menu, uint8_t item, bool draw/*=true*/) {
             break;
           case 3: // Change Filament
             if (draw) {
-              Draw_Menu_Item(row, ICON_ResumeEEPROM, (char*)"Change Filament", true);
+              Draw_Menu_Item(row, ICON_ResumeEEPROM, (char*)"Change Filament");
             } else {
               Popup_Window_ChangeFilament();
               gcode.process_subcommands_now_P(PSTR("M600 B1"));
@@ -1636,6 +1644,18 @@ void Menu_Item_Handler(uint8_t menu, uint8_t item, bool draw/*=true*/) {
             }
           }
           break;
+        #if ENABLED(FILAMENT_LOAD_UNLOAD_GCODES)
+          case 9: // Change Filament
+            if (draw) {
+              Draw_Menu_Item(row, ICON_ResumeEEPROM, (char*)"Change Filament");
+            } else {
+              Popup_Window_ChangeFilament();
+              gcode.process_subcommands_now_P(PSTR("M600 B1"));
+              planner.synchronize();
+              Draw_Print_Screen();
+            }
+            break;
+          #endif
       }
       break;
   }

--- a/Marlin/src/lcd/dwin/e3v2/dwin.cpp
+++ b/Marlin/src/lcd/dwin/e3v2/dwin.cpp
@@ -54,6 +54,10 @@
 
 #if ENABLED(DWIN_CREALITY_LCD)
 
+#if ENABLED(ADVANCED_PAUSE_FEATURE)
+  #include "../../../feature/pause.h"
+#endif
+
 #if ENABLED(HOST_ACTION_COMMANDS)
   #include "../../../feature/host_actions.h"
 #endif
@@ -2079,6 +2083,9 @@ inline void Popup_Control() {
           Draw_Main_Menu();
         }
         break;
+      case M600:
+        Draw_Menu(Prepare, 8);
+        break;
     }
   DWIN_UpdateLCD();
 }
@@ -2090,9 +2097,6 @@ inline void Confirm_Control() {
     switch(popup) {
       case Complete:
         Draw_Main_Menu();
-        break;
-      case M600:
-        Draw_Menu(Prepare, 8);
         break;
     }
   }
@@ -2321,12 +2325,10 @@ inline void HMI_SDCardInit() { card.cdroot(); }
 void MarlinUI::refresh() {}
 
 #if ENABLED(ADVANCED_PAUSE_FEATURE)
-  #include "../../../feature/pause.h"
-
   void MarlinUI::pause_show_message(const PauseMessage message, const PauseMode mode, const uint8_t extruder) {
     // TODO implement remainder of PauseMessage states
     if (message == PAUSE_MESSAGE_INSERT || message == PAUSE_MESSAGE_WAITING) {
-      process = Confirm;
+      process = Popup;
       popup = M600;
       Clear_Screen();
 

